src/utils/interpolation.ts

```diff
@@ -0,0 +1,85 @@
+/* 
+*  This file is based on the work made by DominicVonk at https://github.com/DominicVonk/vscode-variables
+*  This has been imported and updated to allow using the latest @types/vscode library
+*/
+import * as vscode from 'vscode';
+import * as process from 'process';
+import * as path from 'path';
+import { homedir } from 'os';
+
+export function interpolateVariables(value: string | undefined, recursive = false): string | undefined {
+    if (!value) {
```

Comment:
💡 Purely as suggestions, shall we comment for this function for its usage et. al. although if anyone look at the description and other open issues they could understand the borrowed use of this function. Thank you so much for this.

Reply:
Updated, let me know if you want something else to be added